docs/HowToUseSmartFSDumpVisualizer.md

```diff
@@ -9,8 +9,10 @@ There are three main functions in SmartFS Dump Visualizer (SDV).
 
 
 ## How to run SDV?
-To run SDV, please download and install Java to a target PC first.
-[Java Download](https://www.oracle.com/technetwork/java/javase/downloads/jdk12-downloads-5295953.html)
+To run SDV, please download and install Java 8 (or the latest Java) to a target PC first.
+[Java Download](https://www.java.com/ko/download/manual.jsp)
+
+>**Note** To use TizenRT IDE together, Java 8 is recommnaded.
```

Comment:
It is mandatory. not recommended. RT IDE is not working with latest.

Reply:
Okay. Modified as "should be installed".